backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/handlingunits/HUEditorViewFactoryTemplate.java

```diff
@@ -492,6 +492,10 @@ public String getSql(
 
 	private boolean isAlwaysUseSameLayout()
 	{
+		//When the Factory is ServiceHUEditorViewFactory always need to execute the customizeViewLayout method, is not configuration dependent
```

Comment:
* please make the method `protected and override it in `ServiceHUEditorViewFactory`
* add the comment "When the Factory is ServiceHUEditorViewFactory always need to execute the customizeViewLayout method, is not configuration dependent" 
to the overriding method as a javadoc
* (it's cleaner)